packages/reports/src/Containers/reports.tsx

```diff
@@ -88,14 +109,16 @@ const Reports = ({
                                 className='reports__route-selection'
                                 list_items={menu_options().map(option => ({
                                     text: option.label,
-                                    value: option.path,
+                                    value: option.path ?? '',
                                 }))}
-                                value={selected_route.path}
+                                value={selected_route.path ?? ''}
```

Comment:
In `SelectNative` component `list_items` and `value` are required, but in `TRoute` `path` is optional, that is why we pass empty string if `option.path` or` selected_route.path` do not exist.